java-score/src/main/java/com/iconloop/score/pds/PdsPolicy.java

```diff
@@ -191,18 +189,18 @@ public void update_data(String label_id,
         // Verify producer
         String producer = Context.getCaller().toString();
         if (producer_did != null) {
-            DidMessage didMessage = getDidMessage(producer_did, producer_sign);
+            DidMessage didMessage = getDidMessage(producer_did, Context.getCaller(), label_id, "update_data", labelInfo.getLastUpdated(), producer_sign);
             producer = didMessage.did;
-            Context.require(labelInfo.checkNonce(didMessage.nonce), "Invalid Content(LabelInfo) nonce.");
+            Context.require(labelInfo.checkLastUpdated(didMessage.lastUpdated), "Invalid Content(LabelInfo) lastUpdated.");
             Context.require(label_id.equals(didMessage.target), "Invalid Content(LabelInfo) target.");
         }
         if (!labelInfo.getProducer().equals(producer)) {
             Context.revert(101, "You do not have permission.");
         }
 
-        labelInfo.update(null, null, null, null, capsule, data, String.valueOf(Context.getBlockTimestamp()), null, "", null);
+        labelInfo.update(null, null, null, null, capsule, data, null, "", null);
```

Comment:
`created` should not be reset.
```suggestion
        labelInfo.update(null, null, null, null, capsule, data, null, null, null);
```